src/client/app/common/views/pages/room/preview.vue

```diff
@@ -89,6 +90,13 @@ export default Vue.extend({
 			});
 			const objectBoundingBox = new THREE.Box3().setFromObject(obj);
 			this.objectHeight = objectBoundingBox.max.y - objectBoundingBox.min.y;
+
+			const objectWidth = objectBoundingBox.max.x - objectBoundingBox.min.x;
+			const objectDepth = objectBoundingBox.max.z - objectBoundingBox.min.z;
+
+			const horizontal = Math.sqrt(objectWidth ** 2 +  objectDepth ** 2) / camera.aspect;
```

Comment:
```suggestion
			const horizontal = Math.hypot(objectWidth, objectDepth) / camera.aspect;
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/hypot